src/apic-extension/HISTORY.rst

```diff
@@ -3,6 +3,21 @@
 Release History
 ===============
 
+1.1.0
+++++++++++++++++++
+
+**Updates:**
+* Added example for ``az apic api update`` command.
+* Added examples with system assigned identity for ``az apic create`` and ``az apic update`` commands.
+* Added ``--custom-metadata-only`` parameter to ``az apic metadata export`` command.
+
+**Fixes:**
+* Set external document correctly in ``az apic register`` command.
+* [BREAKING CHANGE] Do not use API description as summary in ``az apic register`` command. 
```

Comment:
Let's remove the breaking change label, as PM thinks setting description to summary is a bug.

Reply:
Other part looks good to me. Thaks!

Reply:
ok. thanks~